### PR TITLE
Include stack widget when decoding images and handle null binary exceptions.

### PR DESCRIPTION
**Pull request by Lentumunai-Mark**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes [this issue](https://github.com/onaio/fhir-resources/issues/3091)

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt [hunk 1]
```diff
@@ -734,4 +734,18 @@ class ConfigExtensionsTest : RobolectricTest() {
...
+    val cardViewProperties = profileConfiguration.views[0] as CardViewProperties
+    val listViewProperties = cardViewProperties.content[0] as ListProperties
+    val columnProperties = listViewProperties.registerCard.views[0] as ColumnProperties
+    loadRemoteImagesBitmaps(
+      listOf(columnProperties.children[0]),
+      computedValuesMap = emptyMap(),
+      registerRepository = registerRepository,
+    )
+
+    assertNotNull("Bitmap should be decoded and set correctly")
```
Comment (ellykits) [inline android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt:749]: This assertion will always pass. How is it linked to the test?

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt (hunk 1 above)
Comment (Lentumunai-Mark) [reply android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt:749]: @ellykits  I'll update the test

File: android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt [hunk 2]
```diff
@@ -729,9 +750,108 @@ class ConfigExtensionsTest : RobolectricTest() {
...
+      computedValuesMap = emptyComputedValuesMap,
+      decodedImageMap = decodedImageMap,
+    )
+
+    assertTrue(logMessages.isEmpty())
+    assertTrue(decodedImageMap.isEmpty())
+  }
+
+  @Test
+  fun testExceptionCaughtOnDecodingBitmap() = runTest {
```
Comment (ellykits) [inline android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt:802]: You can pass the expected parameter to the Test annotation to verify that an exception is thrown. e.g. `@Test(expected = NullPointerException.class)`

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt (hunk 2 above)
Comment (Lentumunai-Mark) [reply android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt:802]: @ellykits 
do we have to throw the exception for this particular test?

Comment (ellykits) [reply android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt:789]: That depends on the expected exceptions, null pointer exception was one of them according to your previous implementation.

**Comment by ellykits** [comment]
@Lentumunai-Mark  Delete the unused `decodedBitmap` from `ImageConfig`

**Comment by Lentumunai-Mark** [comment]
noted @ellykits

**Review by ellykits — APPROVED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt [hunk 3]
```diff
@@ -102,6 +103,7 @@ constructor(
 
   val configsJsonMap = mutableMapOf<String, String>()
   val configCacheMap = mutableMapOf<String, Configuration>()
+  val decodedImageMap = mutableMapOf<String, Bitmap>()
```
Comment (ellykits) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt:106]:
```suggestion
  val decodedImageMap = mutableStateMapOf<String, Bitmap>()
```

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt (hunk 3 above)
Comment (Lentumunai-Mark) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt]: noted @ellykits

**Pull request merged**